src/pages/ServiceDetails/__tests__/__snapshots__/ServiceInfo.test.tsx.snap

```diff
@@ -456,6 +456,7 @@ ShallowWrapper {
                           ]
                         }
                         editorLink="/namespaces/istio-system/services/reviews"
+                        validations={undefined}
                       />
```

Comment:
Why validations are undefined? If they can be undefined maybe we shoulld set in the interface this value optional too like destinationrules. What do you think @xeviknal ?

```js
interface ServiceInfoDestinationRulesProps extends EditorLink {
  destinationRules?: DestinationRule[];
  validations: { [key: string]: ObjectValidation };
}
```

Reply:
Changes approached :)